mypy_self_check.ini

```diff
@@ -7,7 +7,6 @@ no_implicit_optional = True
 disallow_any_generics = True
 disallow_any_unimported = True
 warn_redundant_casts = True
-warn_unused_ignores = True
```

Comment:
We want this for mypy itself though. Can you add it back as a package-specific option?
```
[mypy-mypy.*]
<blah>
```

Reply:
I actually *don't* want it for mypy itself—the problems with the linked typeshed PRs are that they make `# type: ignore`s in mypy itself unused, and that leads to test failures in typeshed.